packages/tenant-process/src/services/tenantService.ts

```diff
@@ -0,0 +1,279 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { CreateEvent, eventRepository, initDB } from "pagopa-interop-commons";
+import {
+  Attribute,
+  ExternalId,
+  Tenant,
+  TenantAttribute,
+  TenantEvent,
+  TenantFeature,
+  TenantKind,
+  TenantMail,
+  WithMetadata,
+  tenantAttributeType,
+  tenantEventToBinaryData,
+} from "pagopa-interop-models";
+import { v4 as uuidv4 } from "uuid";
+import { TenantProcessConfig } from "../utilities/config.js";
+import {
+  toCreateEventTenantAdded,
+  toCreateEventTenantUpdated,
+} from "../model/domain/toEvent.js";
+import {
+  ApiInternalTenantSeed,
+  ApiM2MTenantSeed,
+  ApiSelfcareTenantSeed,
+} from "../model/types.js";
+import {
+  invalidAttributeStructure,
+  tenantDuplicate,
+} from "../model/domain/errors.js";
+import {
+  assertAttributeExists,
+  assertExpirationDateExist,
+  assertOrganizationVerifierExist,
+  assertTenantExists,
+  assertVerifiedAttributeExistsInTenant,
+} from "./validators.js";
+import { ReadModelService } from "./readModelService.js";
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export function tenantServiceBuilder(
+  config: TenantProcessConfig,
+  readModelService: ReadModelService
+) {
+  const repository = eventRepository(
+    initDB({
+      username: config.eventStoreDbUsername,
+      password: config.eventStoreDbPassword,
+      host: config.eventStoreDbHost,
+      port: config.eventStoreDbPort,
+      database: config.eventStoreDbName,
+      schema: config.eventStoreDbSchema,
+      useSSL: config.eventStoreDbUseSSL,
+    }),
+    tenantEventToBinaryData
+  );
+  return {
+    async createTenant(
```

Comment:
> this endpoint is not in the tenant process and is outside the scope of the PR. It may be useful to include it, but first it requires some refinements. I suggest to remove this endpoint for now

Sorry, a lot of these unused functions come from their deletion in the PR of `tenant-helpers`, we were going to sync on this soon 😅

Reply:
We probably fixed it